deployer/src/main/java/eu/seaclouds/policy/SeaCloudsInitializerPolicy.java

```diff
@@ -0,0 +1,544 @@
+                            "                  \"links\": []," +
+                            "                  \"nullPointMode\": \"connected\"," +
+                            "                  \"percentage\": false," +
+                            "                  \"pointradius\": 5," +
+                            "                  \"points\": false," +
+                            "                  \"renderer\": \"flot\"," +
+                            "                  \"seriesOverrides\": []," +
+                            "                  \"span\": 12," +
+                            "                  \"stack\": false," +
+                            "                  \"steppedLine\": false," +
+                            "                  \"targets\": [{ " +
+                            "                                    \"column\": \"value\"," +
+                            "                                    \"function\": \"mean\"," +
+                            "                                    \"query\": \"select mean(value) from \\\"" + metricName + "\\\" where $timeFilter group by time($interval) order asc\"," +
+                            "                                    \"refId\": \"A\"," +
+                            "                                    \"series\": \"" + metricName + "\"" +
+                            "                               }]," +
+                            "                  \"timeFrom\": null," +
+                            "                  \"timeShift\": null," +
+                            "                  \"title\": \"" + metricName + "\"," +
+                            "                  \"tooltip\": { \"shared\": true, \"value_type\": \"cumulative\"}," +
+                            "                  \"type\": \"graph\"," +
+                            "                  \"x-axis\": true," +
+                            "                  \"y-axis\": true," +
+                            "                  \"y_formats\": [\"short\",\"short\"]" +
+                            "              }]," +
+                            "      \"title\": \"" + entity.getDisplayName() + "\"" +
+                            "},";
+            rowOptions += currentRow;
+        }
+
+        // Remove last ,
+        rowOptions = rowOptions.substring(0, rowOptions.length() - 1);
+        grafanaInitializerPayload = globalGrafanaOptionsJSON + rowOptions + "] } }";
+
+    }
+
+    @Override
+    public void setEntity(EntityLocal entity) {
+        super.setEntity(entity);
+
+        // SeaCloudsInitializerPolicy should only be attached to Application
+        if (!entity.getApplication().equals(entity)) {
+            //TODO: Check if this exception stops the deployment process
+            throw new RuntimeException("SeaCloudsInitializerPolicy must be attached to an application");
+        }
+
+        parseMonitoringRules();
```

Comment:
The policy is designed to be attached at Application Level. It shouldn't be attached to children entities.